samples/reactjs-todo/client/components/journey/form.js

```diff
@@ -53,15 +52,15 @@ export default function Form() {
         methods.setEmail(user.email);
         methods.setAuthentication(true);
 
-        history.push('/');
+        navigate('/todos');
```

Comment:
I don't think we should change this value. I believe the tutorial expects the user to land on the home page after successfully logging in.

Reply:
Ok sorry i can fix this. lost track of it in some of the changes